podvm/files/etc/systemd/system/kata-agent.service

```diff
@@ -5,9 +5,11 @@ Wants=process-user-data.service
 After=netns@podns.service process-user-data.service
 
 [Service]
-ExecStart=/usr/local/bin/kata-agent --config /etc/agent-config.toml
+# Temporary fix for
```

Comment:
Leftover?

Reply:
Not really, just haven't opened that issue/PR yet.